setup.py

```diff
@@ -1,10 +1,37 @@
 import versioneer
 
 import os
+import sys
+import pkg_resources
 from setuptools import setup, find_packages
 
+
+def numpy_compat(required):
+    new_reqs = [r for r in required if "numpy" not in r and "sympy" not in r]
+    if sys.version_info < (3, 9):
```

Comment:
(3, 9) ?

What is the format?

Reply:
Thats the standard way to check python versions, this check <3.9

Reply:
ok, looks weird